docs/configure

```diff
@@ -701,7 +701,7 @@ virtual_dom_groups:
                                             d.domain_id = g.domain_id and g.enabled = '1' and \
                                             g.name = '${quote_mysql:$local_part}'}}} \
                  {$sender_address} \
-                 {${lookup mysql{select u.username \
+                 {${lookup mysql{select concat_ws('@', localpart, d.domain) \
```

Comment:
please use `u.localpart` instead of just `localpart` where SELECTs use data from multiple tables.